src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -42,7 +42,19 @@ export default class ProjectMetrics extends React.Component<IProjectMetricsProps
     public render() {
         if (this.state.loading) {
             console.log("Waiting for data");
```

Comment:
do we still need this logging statement?

Reply:
no I can remove it